federation/pkg/federation-controller/util/cluster_util.go

```diff
@@ -146,3 +147,23 @@ func GetClientsetForCluster(cluster *federation_v1beta1.Cluster) (*fedclientset.
 	}
 	return nil, err
 }
+
+// Returns a bool to indicate if the object should be forwarded to a cluster
+func SendToCluster(cluster *federation_v1beta1.Cluster, objMeta metav1.ObjectMeta) bool {
```

Comment:
I added a test file for cluster_util.go and a targeted test for this function. I'll expand on the test as we add this functionality object types other than configmap.